clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java

```diff
@@ -417,6 +417,57 @@ public Map reconcile(String cloudProvider, String account, String region, boolea
     return getElasticSearchEntityTagsReconciler().reconcile(this, cloudProvider, account, region, dryRun);
   }
 
+  @Override
+  public Map<String, Object> deleteByNamespace(String namespace, boolean dryRun, boolean deleteFromSource) {
+    List<EntityTags> entityTagsForNamespace = front50Service
+      .getAllEntityTags(false)
```

Comment:
This just fetches the set of entity tags that front50 has in it's in-memory cache ... should be reasonable quick but the response payload is quite large (could be optimized in the future).